torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -120,10 +135,10 @@ PjRtComputationClient::PjRtComputationClient() {
     TF_VLOG(1) << "Initializing PjRt GPU client...";
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
     int local_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
-    std::string dist_service_addr =
-        sys_util::GetEnvString(env::kEnvPjrtDistServiceAddr, "");
+    
+    int global_rank = getGlobalRank(local_rank);
```

Comment:
How about `sys_util::GetEnvInt("RANK", local_rank);`, then we don't need the helper function

Reply:
It's time for me to bring up everyone's favorite topic: TPU v2/v3 multithreading :confetti_ball: 

Normally we could ignore this case entirely since we only needed the distributed runtime for GPUs. But, @jonb377 will be implementing it for TPUs as well, so we have to be more careful about the difference with process rank (shared between threads) and ordinal (unique to each thread) in this context.

Short term, can you make the naming more specific? ie `local_process_rank`, `global_process_rank`, and `getProcessRank` to emphasize these are process-scoped indices?

Reply:
done